pkg/machines/components/vm/stateIcon.jsx

```diff
@@ -0,0 +1,77 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import cockpit from 'cockpit';
+import React, { PropTypes } from "react";
+import {
+    rephraseUI,
+} from "../../helpers.es6";
+
+const _ = cockpit.gettext;
+
+React;
+
+const StateIcon = ({ state, config, valueId, extra }) => {
+    if (state === undefined) {
+        return (<div/>);
+    }
+
+    let stateMap = {
```

Comment:
Nitpick, this could be `const`.

Reply:
it is being assigned to from config.provider bellow